tools/res/cassandra-env.sh

```diff
@@ -0,0 +1,295 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Most of this file is just copied for what is required in the config file. I will add comments where changes have been made.